.github/workflows/test-graphql.yml

```
@@ -18,27 +20,94 @@
+          pnpm run build
+
+      - name: Setup database
+        run: |
+          cd packages/prisma
+          pnpm run prisma:reset:raw -f
+        env:
+          DATABASE_URL: postgresql://klicker:klicker@localhost:5432/klicker-prod
+
+      - name: Generate Hatchet token
+        id: hatchet-token
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:13:24: Double quote to prevent globbing and word splitting

(shellcheck)